content/docs/user-guide/how-to/best-practices.md

```diff
@@ -0,0 +1,136 @@
+# Best Practices for DVC Projects
+
+Asking questions on data science collaboration to data scientists, engineers, or
+managers, we'll get a variety of answers. DVC provides a systematic approach
```

Comment:
No need to keep all the text fro mold versions if it no longer makes sense. This doc has nothing to do with Q&A now so:

```suggestion
DVC provides a systematic approach
```